pywatts/core/step.py

```diff
@@ -173,14 +174,26 @@ def _compute(self, start, end):
             result_dict[key] = res.sel(**{_get_time_indexes(res)[0]: index[(index >= start)]})
         return result_dict
 
-    def _get_target(self, start, batch):
+    def _get_target(self, start, batch, minimum_data=(0, pd.Timedelta(0))):
+        min_data_module = self.module.get_min_data()
+        if isinstance(min_data_module, (int, np.integer)):
+            minimum_data = minimum_data[0] + min_data_module, minimum_data[1]
+        else:
```

Comment:
is it possible that `min_data_module` is an `int` and the argument `minimum_data` is a `pd.Timedelta`, i.e. both if and else should be executed?

Reply:
No, this should not be allowed. The get_min_data in the modules should only return int or timedelta. Mixing this in one module would be confusing I suppose.